src/main/java/com/google/sps/model/GeocodingClient.java

```diff
@@ -26,62 +23,15 @@
  */
 public interface GeocodingClient {
   /**
-   * Sends a GET request to the Google Geocoding API to convert from address to GeocodingResult.
+   * Sends a GET request to the Google Geocoding API to convert from address to GeocodingResult. The
+   * Geocoding API could return multiple results in case of partial matches and in that case, the
+   * first match is the best match hence it would be the result returned. No results would cause a
+   * GeocodingException to be thrown.
    *
    * @param address A String representing the address to geocode.
    * @return A GeocodingResult returned from the Geocoding API.
    * @throws GeocodingException A custom exception is thrown to signal an error pertaining to the
    *     Geocoding API.
    */
   GeocodingResult getGeocodingResult(String address) throws GeocodingException;
```

Comment:
Consider changing the name to indicate it returns the best match of all possible results. Something like `getBestGeocodingResult`.

Reply:
I like this idea a lot, however, it's going to be replaced in the [next PR](https://github.com/googleinterns/blueprint-step-2020/pull/144) with a method that returns `List<GeocodingResult>` which is useful for some of the methods in https://github.com/googleinterns/blueprint-step-2020/pull/144 so unfortunately it wouldn't apply anymore. Thanks though!

Reply:
`RESOLVED`